src/js/index.js

```diff
@@ -0,0 +1,108 @@
+import '../scss/main.scss';
+import UI from './lib/ui';
+import GameBoard from './lib/game_board';
+import AI from './lib/ai';
+
+let gameOver = false;
+UI.generateBoard();
+
+// Place ships
+const humanWaters = GameBoard();
+const computerWaters = GameBoard();
+
+const humanShips = [
+  [15, 16, 17, 18, 19],
+  [48, 58, 68, 78, 88],
+  [22, 23, 24, 25],
+  [60, 61, 62, 63],
+  [31, 32, 33, 34],
+  [97, 98, 99],
+  [1, 2, 3, 4],
+  [80, 90],
+  [75],
+  [55],
+];
+
+const computerShips = [
+  [17, 27, 37, 47, 57],
+  [31, 32, 33, 34, 35],
+  [22, 23, 24, 25],
+  [5, 6, 7, 8],
+  [76, 77, 78, 79],
+  [82, 83, 84],
+  [45, 46, 47],
+  [9, 19],
+  [11],
+  [39],
+];
```

Comment:
[**optional**] Hard coding the positions of the ships will make the game too easy and uninteresting to play. In my second try, I was able to sink all enemy's ship without losing my turn because I knew where all the ships were. So it's best if you randomly place these ships upon every restart of the game.

Reply:
Thanks @JamesOkunlade 

We will make that change later.